src/golang/lib/models/views/artifact_node.go

```diff
@@ -0,0 +1,59 @@
+package views
+
+const ArtifactNodeViewSubQuery = `
```

Comment:
Can you add a comment to the top of these files so describe what it does?

A little confused about what the json_group_array contains. For the outputs json_group_array, does value=all operators that take in this artifact as input and  the inputs json_group_array take in all the operators that create the artifact as output? If so, shouldn't it just be one operator that generates the artifact?

Reply:
Yeah for artifact inputs we could make it unique. In general `json_group_array` operates on `group by` clause and put together elements that are 'grouped' as an json array. We make use of it to put together inputs / outputs list which were separate rows before `group by`.

Reply:
Done, updated query with unique artf input

Reply:
query changes will follow up in #1165